content/docs/1.4/scalers/postgresql.md

```diff
@@ -33,6 +33,7 @@ Finally, a user inserts a query that returns the desired value
 
 - `query:` What query to poll postgresql with. Query must return an integer.
 - `targetQueryValue` - a threshold that is used as `targetAverageValue` in HPA.
+- `metricName` - an optional name to assign to the metric
```

Comment:
Could you please add more information on the metricName and its usage. ie. that KEDA will generate the metric name from xyz values and this setting could be used to override. And that this is important only if there are multiple postresql scalers in one ScaledObject. Thanks